### ISSUE TITLE
Require with rename failed

### ISSUE THREAD

**Issue by maxp** [marcomorain/clojure-lint#15]
The code
```clj
(ns kondo
  (:require
    [clojure.string :refer [blank? join] :rename {join str-join}]))
;
```

Result
```
An unexpected error occured when running clj-kondo. Please report a bug
to https://github.com/marcomorain/clojure-lint/issues
[Error: Invalid arguments
	at new f (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:362:501)
	at new f (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:359:919)
	at range (/home/maxp/.vscode/extensions/marcomorain.clojure-lint-0.6.1/out/extension.js:60:12)
	at toDiagnostic (/home/maxp/.vscode/extensions/marcomorain.clojure-lint-0.6.1/out/extension.js:66:16)
	at Array.map (<anonymous>)
	at /home/maxp/.vscode/extensions/marcomorain.clojure-lint-0.6.1/out/extension.js:92:55
	at /home/maxp/.vscode/extensions/marcomorain.clojure-lint-0.6.1/out/extension.js:36:24
	at ChildProcess.exithandler (child_process.js:310:5)
	at ChildProcess.emit (events.js:200:13)
	at maybeClose (internal/child_process.js:1021:16)
	at Socket.<anonymous> (internal/child_process.js:430:11)
	at Socket.emit (events.js:200:13)
	at Pipe.<anonymous> (net.js:586:12)]
```
